README.md

```diff
@@ -97,12 +97,13 @@ We can think of a `Ref` as of `Effect.Ref`, but with additional functions:
 - `Ref.value :: Ref a -> Dynamic a` to observe the value
 - `Ref.modify :: Ref a -> (a -> a) -> Effect Unit` to modify the value using a function
 
-As a shortcut we have `Ref.set :: Ref a -> a -> Effect Unit` to replace the value completely.
+As a shortcut we have `Ref.write :: Ref a -> a -> Effect Unit` to replace the value completely, 
+and a `Ref.read :: forall m a. MonadEffect m => Ref a -> m a` to read the current value of a `Ref`.
```

Comment:
Inconsistency: let's use either MonadEffect or just Effect, not a mix.

Reply:
NB: it would be fine to have MonadEffect in the code, but Effect in README for easier understanding.